plugins/node/opentelemetry-instrumentation-aws-lambda/test/integrations/lambda-handler.test.ts

```diff
@@ -624,6 +625,37 @@ describe('lambda handler', () => {
       );
       assert.strictEqual(span.parentSpanId, sampledGenericSpanContext.spanId);
     });
+
+    it('creates trace from ROOT_CONTEXT when "disableAwsContextPropagation" is true, eventContextExtractor is provided, and no custom context is found', async () => {
+      process.env[traceContextEnvironmentKey] = sampledAwsHeader;
+      const customExtractor = (event: any): OtelContext => {
+        if (!event.contextCarrier) {
+          return ROOT_CONTEXT;
+        }
+
+        return propagation.extract(context.active(), event.contextCarrier);
+      };
+
+      initializeHandler('lambda-test/async.handler', {
+        disableAwsContextPropagation: true,
+        eventContextExtractor: customExtractor,
+      });
+
+      const testSpan = provider.getTracer('test').startSpan('random_span');
+      await context.with(
+        trace.setSpan(context.active(), testSpan),
+        async () => {
+          await lambdaRequire('lambda-test/async').handler(
+            { message: 'event with no context' },
+            ctx
+          );
+        }
+      );
+
+      const spans = memoryExporter.getFinishedSpans();
+      const [span] = spans;
+      assert.strictEqual(span.parentSpanId, undefined);
```

Comment:
Remind me, what would the parent ID be if we were using active context?

Reply:
It would've been extracted from the propagated context info - see the previous test (`takes sampled custom context over sampled lambda context if "eventContextExtractor" is defined`) for an example.